### DISCUSSION TITLE
What is the correct and/or compatible TERM environment variable value?

### DISCUSSION THREAD

**Discussion by wilx** [microsoft/terminal#14452] [Q&A]
What is the correct and/or compatible TERM environment variable value? Some tools, like Git, needs this set or their complain.

**Comment by chausner** [comment]
Windows Terminal currently does not set the `TERM` environment variable, so there is no corresponding value defined. However, it is possible to define custom environment variables per profile today via the `environment` attribute in your `settings.json`. So if you want to set the variable for compatibility with some other application, it is possible to have it automatically set in Windows Terminal sessions (without the need to set it system-wide).
